WickedEngine/wiRenderPath3D.cpp

```diff
@@ -271,7 +271,7 @@ void RenderPath3D::ResizeBuffers()
 
 		for (uint32_t i = 0; i < desc.mip_levels; ++i)
 		{
-			int subresource_index;
+			uint32_t subresource_index;
```

Comment:
subresource index return is int, why was this changed?

Reply:
same as above, but in this case I made a mistake. The subresource_index has to be signed, because `CreateSubresource` can return a negative number.